CODEOWNERS

```diff
@@ -25,23 +25,12 @@ CODEOWNERS @MrAlias @MadVikingGod @pellared
 detectors/aws/                                                          @open-telemetry/go-approvers @Aneurysm9
 detectors/gcp/                                                          @open-telemetry/go-approvers @dashpole
 
-exporters/autoexport                                                    @open-telemetry/go-approvers @MikeGoldsmith
+exporters/autoexport                                                    @open-telemetry/go-approvers @pellared
 
-instrumentation/github.com/astaxie/beego/otelbeego/                     @open-telemetry/go-approvers
 instrumentation/github.com/aws/aws-lambda-go/otellambda/                @open-telemetry/go-approvers @Aneurysm9
 instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/                   @open-telemetry/go-approvers @Aneurysm9
-instrumentation/github.com/bradfitz/gomemcache/memcache/otelmemcache/   @open-telemetry/go-approvers
-instrumentation/github.com/emicklei/go-restful/otelrestful/             @open-telemetry/go-approvers
 instrumentation/github.com/gin-gonic/gin/otelgin/                       @open-telemetry/go-approvers @hanyuancheung
-instrumentation/github.com/go-kit/kit/otelkit/                          @open-telemetry/go-approvers
-instrumentation/github.com/gocql/gocql/otelgocql/                       @open-telemetry/go-approvers
-instrumentation/github.com/gorilla/mux/otelmux/                         @open-telemetry/go-approvers
-instrumentation/github.com/labstack/echo/otelecho/                      @open-telemetry/go-approvers
-instrumentation/github.com/Shopify/sarama/otelsarama/                   @open-telemetry/go-approvers @pellared @hanyuancheung
```

Comment:
This one is already deprecated per https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4099